tests/flow/base.py

```diff
@@ -25,14 +27,14 @@ def _assert_actual_results_contained_in_expected_results(self,
                                                              num_contained_results):
         actual_result_set = actual_result.result_set[1:]
 
-        # assert num results
+        # Assert num results.
         self.assertEqual(len(actual_result_set), num_contained_results)
 
-        # assert actual values vs expected values
+        # Assert actual values vs expected values.
         expected_result = query_info.expected_result
         count = len([res for res in expected_result if res in actual_result_set])
 
-        # assert number of different results is as expected
+        # Assert number of different results is as expected.
         self.assertEqual(count,
                          num_contained_results,
                          'The actual result is: %s\nThe expected result is: %s' %
```

Comment:
The `_assert_run_time` function in `swilly22/master` reduces time failures to warnings, which would be nice to have here (though it does not need to be part of this specific PR).